src/components/ec-inline-input-field/components/read-only/read-only.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div
+    class="tw-w-full tw-flex tw-justify-between tw-items-center"
+    data-test="ec-inline-input-field-read-only"
+  >
+    <span class="tw-truncate">{{ value }}</span>
+    <button
```

Comment:
make it `type="button"` otherwise it would submit the `<form>` if included in it.